lib/warder/coffee_lint_runner.rb

```diff
@@ -0,0 +1,60 @@
+require 'coffeelint'
+
+module Warder
+  # responsible for run coffeescript style guide validation
+  class CoffeeLintRunner < Runner
+    CLI_OPTION = 'g'
+    CLI_FULL_OPTION = 'coffee-lint'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.